cmd/apps.go

```diff
@@ -85,8 +86,14 @@ func AppsList(results int) error {
 
 	fmt.Printf("=== Apps%s", limitCount(len(apps), count))
 
+	var appNames []string
```

Comment:
One of the examples at https://golang.org/pkg/sort/ has a great example on how to do this: https://play.golang.org/p/ntNak4vvQM :)

> Do you want seperate commits for each action or can I just squash everything together at the end?

Since this kind of enhancement would require a PR at deis/controller-sdk-go for v2, one commit here is fine. Same for the v1 client, which does not require an external library.